src/components/BuildHistoryItem.jsx

```diff
@@ -37,15 +37,20 @@ class BuildHistoryItem extends Component {
         return Math.min((now - start ) / estimation, .95);
     }
 
+    redirectToBuild(slug) {
+        window.location = `https://www.bitrise.io/build/${this}`;
```

Comment:
What do you think about opening the link in a new tab?
```
window.open(`https://www.bitrise.io/build/${this}` _blank);
```

Reply:
Great idea!